src/apps/reducers.js

```diff
@@ -45,8 +45,8 @@ function bundleInfo(state = defaultBundleInfo, action) {
         case FETCH_BUNDLE_ERROR:
             return {
                 bundleId: action.bundleId,
-                bundle: null,
-                inputs: null,
+                bundle: action.bundle,
+                inputs: state.inputs || [],
```

Comment:
also perhaps we should just change inputs default to `[]` an ignore inputs on this one